build-deb.sh

```diff
@@ -1,17 +1,38 @@
 # ------------------------------------------------------------------------------
 #  Copyright (C) 2012, Robert Johansson <rob@raditex.nu>, Raditex Control AB
 #  All rights reserved.
-# 
-#  rSCADA 
+#
+#  rSCADA
 #  http://www.rSCADA.se
 #  info@raditex.nu
-# 
+#
 # ------------------------------------------------------------------------------
 
+# Default settings
+BUILD_BINARY=1
+BUILD_SOURCE=0
+SIGN_KEY=
+
+while [ $# -gt 0 ] && [ "$1" != "--" ]; do
+    case "$1" in
+        --build-source-pkg)
+            BUILD_SOURCE=1
+            ;;
+        --skip-binary)
+            BUILD_BINARY=0
+            ;;
+        --sign-key|-l)
+            SIGN_KEY=$2
+            shift
+            ;;
```

Comment:
This block (lines 17-27) would accept misspelled and random flags.

`./build-deb --foo --bar -mokey 7 --skip-binary --hello --`

No online help/usage.